.npmrc

```diff
@@ -0,0 +1,3 @@
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
+//registry.yarnpkg.com/:_authToken=${NPM_TOKEN}
+@stitch-fix:registry=https://registry.yarnpkg.com/
```

Comment:
This will need to be updated/removed if/when we make our fork of Log Weasel public ... which I'm aiming to.